carpet/loss_gradient.py

```diff
@@ -82,3 +82,11 @@ def synthesis_primal_obj(z, A, L, x, lbda):
     cost = 0.5 * np.sum(np.square(z.dot(L).dot(A) - x))
     cost += lbda * np.sum(np.abs(z[:, 1:]))
     return cost / n_samples
+
+
+def loss_prox_tv_analysis(x, u, lbda):
```

Comment:
Will this be used elsewhere than in `compute_prox_tv_errors`?

Reply:
It could. It is the general loss for `prox_tv` so I think this is the right place to put it.